src/test/javascript/spec/component/overview/course-discussion/course-discussion.component.spec.ts

```diff
@@ -129,56 +131,34 @@ describe('CourseDiscussionComponent', () => {
     it('should initialize formGroup correctly', fakeAsync(() => {
         component.ngOnInit();
         tick();
-        expect(component.formGroup.get('context')?.value).toEqual({
-            courseId: metisCourse.id,
-            courseWideContext: undefined,
-            exerciseId: undefined,
-            lectureId: undefined,
-            searchText: undefined,
-            filterToUnresolved: false,
-            filterToOwn: false,
-            filterToAnsweredOrReacted: false,
-            postSortCriterion: PostSortCriterion.CREATION_DATE,
-            sortingOrder: SortDirection.DESCENDING,
-        });
+        expect(component.formGroup.get('context')?.value).toEqual([]);
         expect(component.formGroup.get('sortBy')?.value).toBe(PostSortCriterion.CREATION_DATE);
         expect(component.formGroup.get('filterToUnresolved')?.value).toBeFalse();
         expect(component.formGroup.get('filterToOwn')?.value).toBeFalse();
         expect(component.formGroup.get('filterToAnsweredOrReacted')?.value).toBeFalse();
+        expect(component.currentSortDirection).toBe(SortDirection.DESCENDING);
```

Comment:
This now tests a bit fewer variables. Is this intended?

Reply:
Yes, since the 'context' element within the formGroup now contains a list of selected elements, which is empty in the beginning. That the other variables are set correctly is checked by test case above.